main.cpp

```diff
@@ -13,75 +13,506 @@ What you create in this project will be used as the basis of Project 5 in the co
+    4) condiment used // string
+    5) number of calories // int
+3 things it can do:
+    1) cause foodpoisoning
+    2) provide a satisfying meal
+    3) go moldy if left uneaten
+ */
+
+struct Sandwich
+{
+    // type of bread
+    std::string bread = "brown";
+    // type of fillings
+    std::string fillings = "meat";
+    // amount of different fillings
+    int numOfFillings = 2;
+    // condiment used
+    std::string condiment = "mayo";
+    // number of calories
+    int calories = 600;
+
+    // cause foodpoisoning
+    void foodPoisoning(std::string fillings = "uncooked chicken");
+    // provide a satisfying meal
+    void satisfyingMeal(int calories);
+    // go moldy if left uneaten
+    void moldy(int daysWithoutStorage);
+};
+
+/*
+Thing 3) Soccer Team
+5 properties:
+    1) number of players in squad // int
+    2) average age of players in squad // int
+    3) number of total goals scored by the team // int
+    4) number of matches in the season // int
+    5) league position // int
+3 things it can do:
+    1) train their skills
+    2) win a game
+    3) move up the league table
+ */
+
+struct SoccerTeam
+{
+    class Manager
+    {
+        std::string name = "Pep Gaurdiola";
+        int Age = 48;
+        std::string birthPlace = "Spain";
+        int yearsActive = 15;
+        int numberOfTeamsManaged = 3;
+
+        void trainTeam(std::string nameOfCoach, bool deligateTrainingSession);
+        void signNewPlayers(std::string market, int age, std::string playerPosition);
+        void speakToThePress(bool attendPressConference = true);
+    };
+
+    // number of players in squad
+    int squadPlayers = 24;
+    // average age of players in squad
+    int averageAge = 28;
+    // number of total goals scored by the team
+    int totalGoalsScored = 60;
+    // number of matches in the season
+    int numOfMatchesToPlay = 38;
+    // league position
+    int leaguePosition = 4;
+
+    // train their skills
+    void trainingSession(Manager managerA);
+    // win a game
+    void winMatch(bool motivated);
+    // move up the league table
+    void changeLeaguePosition(int pointsAccquired);
+};
+
+/*
+Thing 4) Synth
+5 properties:
+    1) number of oscillators // int
+    2) amount of presets // int
+    3) number of filters // int
+    4) number of outputs // int
+    5) type of built-in effect // string
+3 things it can do:
+    1) output midi
+    2) filter out all frequences above 10khz
+    3) load different presets
+ */
+
+struct Synth
+{
+    // number of oscillators
+    int oscillators = 3;
+    // amount of presets
+    int presets = 150;
+    // number of filters
+    int filters = 3;
+    // number of outputs
+    int outputs = 2;
+    // type of built-in effect
+    std::string effectType = "delay";
+
+    // output midi
+    void outputMidi(int outputs);
+    // filter out all frequences above 10khz
+    void highCut10khz(bool filterSweepDown = true, bool adjustQFactor = true);
```

Comment:
'high-cut' is not a very good action term.   
a better name here would be `filterAllFrequenciesAbove10k`
- [ ] revise